Adyen/Model/PrefilledShopperInformation.swift

```diff
@@ -6,6 +6,20 @@
 
 import Foundation
 
+public struct PhoneNumber {
+
+    /// The phone number.
+    public let value: String
+
+    /// The phone extension.
```

Comment:
```suggestion
    /// The phone extension in E.123 international notation (ex. +22).
```

Reply:
According to E.123 this would be called the `country code` as far as I understand. So maybe good to also rename it from `phoneExtension` to `countryCode`

Reply:
I am concerned if this could be interpreted as ISO 3166-1 "country code" (ex. "NL", "US"), since in other places we  request it..

Reply:
We can use ["Country calling codes"](https://en.wikipedia.org/wiki/List_of_country_calling_codes) or ["telephone prefixes"](https://en.wikipedia.org/wiki/List_of_mobile_telephone_prefixes_by_country)

Reply:
Ok I see we already have a `PhoneExtension` that has a `countryCode` so indeed it could be misleading. Maybe `callingCode` would be enough as `internationalCallingCode` might be a bit too long 🙃 
https://en.wikipedia.org/wiki/List_of_mobile_telephone_prefixes_by_country

Reply:
I have renamed the "phoneExtension" to "callingCode" which makes sense :) Thanks for the suggestion guys :)

Reply:
af7e655da06c724c19c3990ffd44e3c7dc8cf38a